modules/core/04-channel/v2/keeper/keeper.go

```diff
@@ -261,3 +261,24 @@ func (k *Keeper) getAllPacketStateForClient(ctx sdk.Context, clientID string, pr
 	}
 	return packets
 }
+
+// SetBaseClient sets the base client ID under the base client key for an alias
+func (k *Keeper) SetBaseClient(ctx sdk.Context, alias string, baseClientID string) {
```

Comment:
Could this just be named something like "SetClientAlias"?

Reply:
It took me a little time to realize what baseClient actually was, and having alias in the function name might be it clearer while reading the usage of this

Reply:
Yes, though the key here is the alias, and the value is the underlying clientID. So i feel it would be confusing to name it setting alias since that is not what the value is. But open to changing